stubs/Helpers.stub

```diff
@@ -131,3 +131,32 @@ function tap($value, $callback = null)
 function view($view = null, $data = [], $mergeData = [])
 {
 }
+
+/**
+ * Translate the given message.
+ *
+ * @param  string|null  $key
+ * @param  array<string, string>  $replace
+ * @param  string|null  $locale
+ * @return ($key is null ? \Illuminate\Contracts\Translation\Translator : mixed)
+ */
+function trans($key = null, $replace = [], $locale = null)
+{
+    if (is_null($key)) {
+        return app('translator');
+    }
+
+    return app('translator')->get($key, $replace, $locale);
```

Comment:
This is a stub. You don't need function bodies.

Reply:
It was still a WIP to show my attempt :)
Fixed now